runtime/plaid-stl/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "plaid_stl"
-version = "0.27.1"
+version = "0.28.0"
```

Comment:
Technically this is backwards compatible so could be 27.2.

I don't feel strongly though.